dev-haskell/ghc-lib/ghc-lib-9.8.4.20241130.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.9.0.0.9999
+
+CABAL_HACKAGE_REVISION=1
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour"
+inherit haskell-cabal
+
+DESCRIPTION="The GHC API, decoupled from GHC versions"
+HOMEPAGE="https://github.com/digital-asset/ghc-lib"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+IUSE="+threaded-rts"
```

Comment:
Rename this to `threads`, which is more canonical, please.

```
hackport merge --flags=threaded-rts:threads 'ghc-lib-9.8.4.20241130'
```

Hopefully it updates `metadata.xml` too, but double check that when you're done.